### DISCUSSION TITLE
Monthly view when answering a poll

### DISCUSSION THREAD

**Discussion by Jaakkonen** [lukevella/rallly#694] [Ideas]
In use cases where you want to find longest period of days that are ok for a group of people (for instance a vacation abroad) you often have easily 70 days. 

Laid out horizontally it's really hard to cross-check the options with ones own calendar
![image](https://github.com/lukevella/rallly/assets/33450373/3bed44a7-481f-4658-8346-0f2d839e6b9b)

whereas with a monthly view it's much simpler like in the poll creation tool
![image](https://github.com/lukevella/rallly/assets/33450373/9ee99cb3-a097-4f67-9c64-bd962d8c8ad4)

It'd be super nice to have the options on a calendar (monthly view) and being able to choose those from there while answering!

**Comment by lukevella** [comment]
I think what you're looking for is not a meeting poll which is what Rallly currently offers but what I would call an "availability heat-map". This is the kind of solution that crab.fit and when2meet.com offers. Rather than slapping extra features to the meeting poll to make it work like a heat-map I would prefer to keep these two solutions distinct from each other. I do want to offer this solution as part of Rallly and I see these two solutions working hand in hand with each other. So a possible workflow would be, start off with a heat-map, select a few slots from the heat-map to create a meeting poll and then finalize a date.

**Comment by ScyllPoesis** [comment]
Just chiming in to see if this is still something that you consider part of the roadmap. I personally don't think it's a major feature deviation as in my eyes this is just a different representation of the same data; instead of showing a list that scrolls to the side, it's just a calendar view with the same info given to the user (colored status icon, number of 'yes'es). Rallly has such a nice design and responsiveness to it, and I think just having a view toggle for cases can be helpful for more organic voting, especially when days skip/jump around a bit, and to keep the UX simple only allow this view for polls that don't have individual time slots specified.

Cheers!
